app/component/SummaryRow.js

```diff
@@ -267,7 +267,7 @@ const SummaryRow = (
     if (leg.intermediatePlace) {
       legs.push(<ViaLeg key={`via_${leg.mode}_${leg.startTime}`} />);
       if (
-        (noTransitLegs && isThresholdMet) ||
+        isThresholdMet ||
```

Comment:
is changing the code required or is noTransitLegs a configurable property which just should be enabled?

Reply:
We can comment out the section where noTransitLegs variable is set to false (by default it's true) but I don't think that we could manage to change it's value from outside.